examples/places-service-with-element/components/places-service-with-element/places-service-with-element.tsx

```diff
@@ -0,0 +1,66 @@
+import React, {useEffect, useRef, useState} from 'react';
+
+import {useGoogleMap, usePlacesService} from '@ubilabs/google-maps-react-hooks';
+
+import styles from './places-service-with-element.module.css';
+
+const PlacesServiceElement = () => {
+  const map = useGoogleMap();
+
+  const divRef = useRef<HTMLDivElement>(null);
+
+  const [placeResults, setPlaceResults] = useState<
+    google.maps.places.PlaceResult[]
+  >([]);
+
+  // Get the places service from the usePlacesService hook
+  const service = usePlacesService({divElement: divRef.current});
```

Comment:
Is this working? 🤔  The change of the `divRef` value doesn't trigger rendering, so I would recommend to use a state for the div element as it's done for the map container.

Reply:
Okay, it gets rerendered because of another reason. Will adjust it  👍